docs/aca/09-aca-autoscale-keda/index.md

```diff
@@ -103,27 +105,28 @@ az containerapp update `
 --scale-rule-auth "connection=svcbus-connstring" `
 --scale-rule-metadata "topicName=<Your topic name>" `
                         "subscriptionName=<Your topic subscription name>" `
-                        "namespace=<Your service bus namespace>" `
+                        "namespace=$NamespaceName" `
                         "messageCount=10" `
                         "connectionFromEnv=svcbus-connstring"
 ```
 
 What we have done is the following:
 
-* Setting the minimum number of replicas to `1`, means that this Container App could be scaled-in to one single replica if there are no new messages on the topic.
-* Setting the maximum number of replicas to `5`, means that this Container App will not exceed more than 5 replicas regardless of the number of messages on the topic.
-* Setting a friendly name for the scale rule `topic-msgs-length` which will be visible in Azure Portal.
-* Setting the scale rule type to `azure-servicebus`, this is important to tell KEDA which type of scalers our Container App is configuring.
-* Setting the authentication mechanism to type `connection` and indicating which secret reference will be used, in our case `svcbus-connstring`.
-* Setting the `metadata` dictionary of the scale rule, those matching the metadata properties in KEDA template we discussed earlier.
-* Disabled the integration with SendGrid as we are going to send load of messages now to test the scale out rule.
+* Setting the minimum number of replicas to `1`. This means that this Container App could be scaled-in to a single replica if there are no new messages on the topic.
+* Setting the maximum number of replicas to `5`. This means that this Container App will not exceed more than 5 replicas regardless of the number of messages on the topic.
+* Setting a friendly name for the scale rule `topic-msgs-length` which will be visible in the Azure Portal.
+* Setting the scale rule type to `azure-servicebus`. This is important to tell KEDA which type of scalers our Container App is configuring.
+* Setting the authentication mechanism to type `connection` and indicating which secret reference will be used. In our case `svcbus-connstring`.
+* Setting the `metadata` dictionary of the scale rule. Those match the metadata properties in KEDA template we discussed earlier.
+* Disabled the integration with SendGrid as we are going to send several messages to test the scale out rule.
+
+**Note About Setting Minimum Replicas To 0:**
```

Comment:
maye be move this under !!! note?

Reply:
Also the bullet seems a bit off in preview page. 
![image](https://user-images.githubusercontent.com/8058415/229438500-4353b845-51bf-497d-92c4-d746d30c17ea.png)